contrib/endpoints/src/api_manager/quota_control.cc

```diff
@@ -46,7 +46,10 @@ void QuotaControl(std::shared_ptr<context::RequestContext> context,
         TRACE(trace_span) << "Quota service control request returned with "
                           << "status " << status.ToString();
 
-        continuation(status);
+        // if the quota control service API is not available, then pass client
+        // requests
```

Comment:
I see. could you update the comment
// quota control is using "failed open" policy.  If the server is not available, allow the request to go.